### PR TITLE
Garden Waste Cancellation improvments

### PR DESCRIPTION
**Pull request by struan**
Fill in some missing code, plus be a bit more helpful about pending cancellations, and subscriptions

[skip changelog]

### CODE REVIEW

File: templates/web/base/waste/bin_days.html [hunk 1]
```diff
@@ -91,7 +94,7 @@ <h3>More services</h3>
             [% IF any_request_allowed %]
               <li><a href="[% c.uri_for_action('waste/request', [ property.id ]) %]">Request a new container</a></li>
             [% END %]
-            [% IF !services_available.Garden_Waste.is_active %]
+            [% IF !pending_subscription OR !services_available.Garden_Waste.is_active %]
```
Comment (dracos) [inline templates/web/base/waste/bin_days.html:97]: Should this be an AND?

File: templates/web/base/waste/garden/cancel_confirmation.html [hunk 2]
```diff
@@ -4,6 +4,13 @@ <h1 class="govuk-heading-xl">Subscription cancelled</h1>
 
 <p>Any needed text here</p>
 
+[% IF payment_method == 'direct_debit' %]
+<p>
+It will take a few days to confirm the cancellation of your direct debit so you will
+see a Garden Waste Collection on your bin calendar untill that happens.
```
Comment (dracos) [inline templates/web/base/waste/garden/cancel_confirmation.html:10]: s/untill/until

File: perllib/FixMyStreet/App/Controller/Waste.pm [hunk 3]
```diff
@@ -96,6 +96,26 @@ sub check_payment_redirect_id : Private {
     $c->stash->{report} = $p;
 }
 
+sub get_pending_subscription : Private {
+    my ($self, $c) = @_;
+
+    my $uprn = $c->stash->{property}{uprn};
+    my $len = length $uprn;
+    my $subs = $c->model('DB::Problem')->search({
```
Comment (dracos) [inline perllib/FixMyStreet/App/Controller/Waste.pm:104]: Restrict to body?

Comment (dracos) [reply perllib/FixMyStreet/App/Controller/Waste.pm:104]: This also feels a bit fragile, but not sure I have anything to suggest.

**Review by dracos — CHANGES_REQUESTED** [review]
Couple of small queries.

File: perllib/FixMyStreet/App/Controller/Waste.pm (hunk 3 above)
Comment (struan) [reply perllib/FixMyStreet/App/Controller/Waste.pm:104]: Yes, it's not ideal.

**Review by dracos — APPROVED** [review]

**Pull request merged**